var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -475,7 +476,7 @@ def cmake_args(self):
 
         if self.spec.satisfies("+wrapper"):
             options.append(
-                self.define("CMAKE_CXX_COMPILER", self.spec["kokkos-nvcc-wrapper"].kokkos_cxx)
+                self.define("CMAKE_CXX_COMPILER", self["kokkos-nvcc-wrapper"].kokkos_cxx)
```

Comment:
```suggestion
                self.define("CMAKE_CXX_COMPILER", self["kokkos"].kokkos_cxx)
```

If we can exploit the property defined above (Only one place to maintain the logic).

Reply:
Done, we can even delete the subscript, and just use self.